deployment/deployment.json

```diff
@@ -2757,7 +2757,7 @@
         "status": "dev",
         "versions": {
           "schema": "1.3.0",
-          "subgraph": "1.1.2",
+          "subgraph": "1.1.5",
```

Comment:
I only see version updates for Arbitrum and Ethereum. Is that all that is impacted by these changes?

Reply:
Whoops, arbitrum wasn't intended to be changed. 

I still have to deploy on all the other networks, so far only ethereum has been deployed and QAd

Reply:
SG